### ISSUE TITLE
Sort many photosets with 'flickr.photosets.orderSets' fails (URI too long)

### ISSUE THREAD

**Issue by rokrupnik** [Pomax/node-flickrapi#108]
When I tried to change the order of my photosets using your package I ran into a problem, because I had too many photosets and therefore the URI of request was too long.

Try to use POST requests for such methods. If it's any help take a look at the same issue in a python flickrapi implementation:
https://github.com/sybrenstuvel/flickrapi/issues/74

**Comment by Pomax** [comment]
What do you mean with "too many photosets"? The photoset responses should be pages, like any other response, so the number of sets should not matter (I have 50+ sets myself and have not run into issues there yet)

**Comment by rokrupnik** [comment]
I think you missunderstood me - I'm not talking about retrieving the photosets, but ordering them using the endpoint `flickr.photosets.orderSets`. I have 100-200 photosets and when I try to order them, the URI is too long, because all the ids go in the query parameter.

**Comment by Pomax** [comment]
looking at https://www.flickr.com/services/api/flickr.photosets.orderSets.html, this should already be a POST (not a GET) so that's more a bug than a feature request I guess.

**Comment by rokrupnik** [comment]
It may be so. I looked inside your code and I saw no way to make this a
POST request by default. I tried to fix it myself but I ran into problems
with the signing procedure.

**Comment by Pomax** [comment]
yeah there are no provisions for POST atm, the API parser will need an update to discover that certain methods *require* POST rather than GET, with the query function updated to check for the method and falling through to either a `getQuery`, with the current code, or a `postQuery`, which would form a simple signed URL with a post payload representing the required arguments for the call to return a proper result.
